main.js

```diff
@@ -0,0 +1,11 @@
+const bars = document.getElementById('pop');
+const closeMenu = document.querySelector('.close');
+const popUp = document.querySelector('.menu-up');
+
+bars.addEventListener('click', () => {
+  popUp.classList.add('pop-up');
+});
+
+closeMenu.addEventListener('click', () => {
+  popUp.classList.remove('pop-up');
+});
```

Comment:
- [ ] Good job on the functionality but there is a little issue according to the requirements, When the user clicks any of the mobile menu options, the mobile menu should disappear. In your project when I click on any of the mobile menu options the mobile menu doesn't disappear. Please fix this issue
  - To fix this you can add events in your javascript file, When any of the mobile menu options is clicked you can set mobile menu `display: none;`

### See the video for reference 

https://user-images.githubusercontent.com/83803414/161627952-7ce84874-d31f-4640-a0df-4ae089260f98.mp4